### ISSUE TITLE
Color contrast for Community Care Service Type field hint text

### ISSUE THREAD

**Issue by r-bartlett-gsa** [department-of-veterans-affairs/va.gov-cms#22913]
## Open questions

- Does it make sense to combine with https://github.com/department-of-veterans-affairs/va.gov-cms/issues/22888?

---
## Describe the defect

> **Issue: Hint text doesn’t pass color contrast requirements**
> **Field:** Community Care Service Type
> **Expected severity level:** 1
> **Expected experience standard:** User can perceive all meaningful elements
> **Description:** The hint text under “Service type” doesn’t pass color contrast requirements. Some users won’t be able to read this text:
> **How to recreate:** Use a color contrast checker (like [Colour Contrast Analyzer](https://www.tpgi.com/color-contrast-checker/))
> **Recommendation:** Change the hint text color from --vads-color-gray-medium to --vads-color-gray (or #3D4551, same as autosuggest field #22477 ), which passes.

The default hint text assumes a white background, but our form uses a gray background. Either the foreground or background color needs to change.

[VADS hint text component](https://design.va.gov/components/form/#hint-text)
[MDN color contrast guide](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Perceivable/Color_contrast)

## To Reproduce
1. Go to facility locator on prod
2. In the 'Facility type' field, select 'Community Care'. This will display the 'Service type' field.
3. Run a color contrast checker of your choice.

## Screenshots
<img width="344" height="299" alt="Hint text and error report showing 4.04 contrast" src="https://github.com/user-attachments/assets/86b1bf3f-f54b-488c-bdc0-f68ddf8aaee2" />

 ## AC / Expected behavior

- [ ] The hint text no longer throws a contrast error. Color contrast must meet a minimum of 4.5:1.

## Definition of done:
- [ ] Code completed; PR created
- [ ] Review instance created/available; internal team notified to start reviews
- [ ] Code review (meets established quality and security standards)
- [ ] Design review (meets design & UX requirements)
- [ ] A11y review (meets accessibility standards)
- [ ] Functional/feature review (meets acceptance criteria)
- [ ] Automated tests passed; PR approved
- [ ] Testing completed; dev team notified to deploy
- [ ] PO/PM/external verify in prod
- [ ] Documentation updated

**Comment by carlyMhayter** [comment]
Although this is a really simple change to make, I am currently blocked because the last three review instances I have generated to verify this change have both failed :C See the [ticket I opened in the VA Platform Support channel](https://dsva.slack.com/archives/CBU0KDSB1/p1768938269296449). @r-bartlett-gsa

**Comment by r-bartlett-gsa** [comment]
@carlyMhayter It appears that the color contrast is already changed for this field on staging. Can you verify if this was done as part of this PR: https://github.com/department-of-veterans-affairs/vets-website/pull/41310 If so this can be closed as completed.

**Comment by carlyMhayter** [comment]
Yes, that does appear to fix the issue!
